.dockerignore

```diff
@@ -0,0 +1,3 @@
+*
```

Comment:
This scares me 😅 Are we sure we can ignore everything? Do I miss-understand it?

Reply:
This ignores everything but the following lines negate it to include the required files, so all files in `lib` and any filename beginning with `Gemfile` will be available in the build context but nothing else